pkgs/development/python-modules/setuptools-rust/default.nix

```diff
@@ -21,7 +25,15 @@ buildPythonPackage rec {
 
   nativeBuildInputs = [ setuptools-scm ];
 
-  propagatedBuildInputs = [ semantic-version setuptools typing-extensions ];
+  propagatedBuildInputs = [ semantic-version setuptools typing-extensions rustc cargo ];
+
+  setupHook = ./setuptools-rust-hook.sh;
+  pyLibPrefix = python.libPrefix;
+  cargoBuildTarget = "${targetPackages.rust.toRustTargetSpec targetPackages.stdenv.hostPlatform}";
```

Comment:
```suggestion
  cargoBuildTarget = targetPackages.rust.toRustTargetSpec targetPackages.stdenv.hostPlatform;
```

Reply:
Yeah... the string only complicates it here.